src/screens/ExchangeModal.tsx

```diff
@@ -1,916 +0,0 @@
-import { useRoute } from '@react-navigation/native';
-import lang from 'i18n-js';
-import { isEmpty, isEqual } from 'lodash';
-import React, { MutableRefObject, useCallback, useEffect, useLayoutEffect, useMemo, useRef, useState } from 'react';
-import equal from 'react-fast-compare';
-import { EmitterSubscription, InteractionManager, Keyboard, NativeModules, TextInput, View } from 'react-native';
-import { useAndroidBackHandler } from 'react-navigation-backhandler';
-import { useDispatch } from 'react-redux';
-import { useDebounce } from 'use-debounce';
-import { useMemoOne } from 'use-memo-one';
-import { dismissingScreenListener } from '../../shim';
-import {
-  ConfirmExchangeButton,
-  ExchangeDetailsRow,
-  ExchangeFloatingPanels,
-  ExchangeHeader,
-  ExchangeInputField,
-  ExchangeNotch,
-  ExchangeOutputField,
-} from '../components/exchange';
-import { FloatingPanel } from '../components/floating-panels';
-import { GasSpeedButton } from '../components/gas';
-import { KeyboardFixedOpenLayout } from '../components/layout';
-import { delayNext } from '../hooks/useMagicAutofocus';
-import { getRemoteConfig, useRemoteConfig } from '@/model/remoteConfig';
-import { WrappedAlert as Alert } from '@/helpers/alert';
-import { analytics } from '@/analytics';
-import { Box, Row, Rows } from '@/design-system';
-import { GasFee, LegacyGasFee, LegacyGasFeeParams, SwappableAsset } from '@/entities';
-import { ExchangeModalTypes, isKeyboardOpen } from '@/helpers';
-import { KeyboardType } from '@/helpers/keyboardTypes';
-import { getFlashbotsProvider, getProvider } from '@/handlers/web3';
-import { delay, greaterThan } from '@/helpers/utilities';
-import {
-  useAccountSettings,
-  useColorForAsset,
-  useGas,
-  usePrevious,
-  usePriceImpactDetails,
-  useSwapCurrencies,
-  useSwapCurrencyHandlers,
-  useSwapDerivedOutputs,
-  useSwapInputHandlers,
-  useSwapInputRefs,
-  useSwapIsSufficientBalance,
-  useSwapSettings,
-  useWallets,
-} from '@/hooks';
-import { loadWallet } from '@/model/wallet';
-import { useNavigation } from '@/navigation';
-import { walletExecuteRap } from '@/raps/execute';
-import { swapClearState, SwapModalField, TypeSpecificParameters, updateSwapSlippage, updateSwapTypeDetails } from '@/redux/swap';
-import { ethUnits } from '@/references';
-import Routes from '@/navigation/routesNames';
-import { ethereumUtils, gasUtils } from '@/utils';
-import { IS_ANDROID, IS_IOS, IS_TEST } from '@/env';
-import { logger, RainbowError } from '@/logger';
-import { CROSSCHAIN_SWAPS, useExperimentalFlag } from '@/config';
-import { CrosschainQuote, Quote, SwapType } from '@rainbow-me/swaps';
-import store from '@/redux/store';
-import { getCrosschainSwapServiceTime } from '@/handlers/swap';
```

Comment:
this can be removed from `handlers/swap` as it is no longer used: `getCrosschainSwapServiceTime`